app/retail/templates/bounties/contributor/story.html

```diff
@@ -38,8 +38,16 @@
     </div>
     <div class="col-12 col-md-5 offset-md-2 my-auto mt-4 order-md-2 g-fadein" data-fade-duration=1000 data-fade-direction="right">
       <p>
-        There are open issues in our Issues Explorer for all Python, JavaScript, Rust, Go, Solidity,
-        Node.js, and many more developers.
+        There are open issues & bug bounties in our <a href="/explorer">Issue Explorer</a> for 
```

Comment:
Please use `{% url 'explorer' %}` instead of relative URLs here. And  similarly for all the other links. Thanks.